keras/optimizers.py

```diff
@@ -274,13 +274,16 @@ def get_config(self):
 
 
 class Adagrad(Optimizer):
-    """Adagrad optimizer.
+    """Adagrad optimizer. Adagrad is a gradient-based optimizer
```

Comment:
The first line (docstring summary) should be exactly one line. Introduce 2 line breaks after the first period